plugins/node/opentelemetry-instrumentation-aws-lambda/src/instrumentation.ts

```diff
@@ -132,7 +132,7 @@ export class AwsLambdaInstrumentation extends InstrumentationBase {
               return moduleExports;
             },
             (moduleExports?: LambdaModule) => {
-              if (moduleExports == undefined) return;
+              if (moduleExports === undefined) return;
```

Comment:
```suggestion
              if (moduleExports == null) return;
```
To keep behavior as before